src/datapump/sync/sync.py

```diff
@@ -407,7 +407,7 @@ class RADDAlertsSync(DeforestationAlertsSync):
     source_bucket = "gfw_gee_export"
     source_prefix = "wur_radd_alerts/"
     input_calc = "(A >= 20000) * (A < 40000) * A"
-    number_of_tiles = [174, 175]
+    number_of_tiles = [190, 191]  # Africa:54, Asia:52, CA: 16, SA:68
```

Comment:
Have we already discussed making this automagic? I _think_ we have. But I may be confusing it with another conversation.

Reply:
Could you clarify what is meant by automagic? :)
The problem this is trying to solve is a lot of the time we find a new version in Google Cloud Storage while it is in the middle of being uploaded. We don't want to import an incomplete dataset, which would cause tiles to vanish from the map, so we make sure that the set is complete by waiting until the expected number of tiles are present. I'm not crazy about it, but we rarely have to change it. I'm all-ears for a better solution. :)

Reply:
Thanks for the reminder around our current process, @dmannarino. I just said _automagic_ since I was foggy about the details for getting the updated dataset. If it's really infrequent, then maybe it's not worth the trouble to think of another update process.